airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -301,36 +301,39 @@ def create_custom_object(
     ):
         """
         Creates custom resource definition object in Kubernetes
-
         :param group: api group
         :param version: api version
         :param plural: api plural
         :param body: crd object definition
         :param namespace: kubernetes namespace
         """
         api = client.CustomObjectsApi(self.api_client)
+
         if namespace is None:
             namespace = self.get_namespace()
-        if isinstance(body, str):
-            body_dict = _load_body_to_dict(body)
-        else:
-            body_dict = body
```

Comment:
again here is a completely unrelated change that is purely stylistic.  in this situation i think it's best to just leave it alone.  oneline vs if / else is the stylistic choice of the original developer, and it's perfectly acceptable. meanwhile, every change has risk of introducing bug and it's also a distraction for the reviewer.

Reply:
I would like to object here. In such ternary operations the special construction is a good style. Also we need type hint, which in its original form will be not good:
`
         if isinstance(body, str):
            body_dict: dict = _load_body_to_dict(body)
        else:
            body_dict: dict = body
`

Reply:
same here, as you wish

Reply:
The rule of thumb here is that if possible - such changes should be separted out. This makes it easier for cherry-picking (in case we decide to cherry-pick your change to 2.4 branch it might cause unnecessary conflicts that we will have to resolve. So I suggest you split that change out into a separate PR.

This is basically the rule that everyone shoudl follow  - when you make a bugfix or change, avoid doing unrelated refactoring and separate it out to a separate PR. We are continiuously doing it and while it des not always work, it is something we strive for. 

This is much more important for Airlfow's and Core provider's code - because it allows cherry-picking and produces much better changelogs (we use commit messages to automatically generate changelog. So this is rather justified requirement that has very good reasons and I kindly ask  you to follow that.

Reply:
OK, thank you for comments
I see how high communications skills required from you to rule this project =)
I do appreciate this